src/index.md

```diff
@@ -483,7 +479,7 @@ To recursively call a makefile, use the special `$(MAKE)` instead of `make` beca
 new_contents = "hello:\n\ttouch inside_file"
 all:
 	mkdir -p subdir
-	echo $(new_contents) | sed -e 's/^ //' > subdir/makefile
+	echo -e $(new_contents) | sed -e 's/^ //' > subdir/makefile
```

Comment:
Ah interesting! I think what we want here is `printf` instead of `echo` or `echo -e`.

I just learned that echo is super inconsistent.. for example on my machine `echo -e` doesn't work.
More details: https://unix.stackexchange.com/questions/65803/why-is-printf-better-than-echo/65819#65819